modules/private-clusters-about-azure.adoc

```diff
@@ -14,7 +14,7 @@ The cluster still requires access to Internet to access the Azure APIs.
 
 The following items are not required or created when you install a private cluster:
 
-* A `BaseDomainResourceGroup`, since the cluster does not create public records
+* A base domain resource group, since the cluster does not create public records
```

Comment:
I wonder if this has specific significance to Azure, and should remain code font?

Reply:
Yeah, this is Azure related; I'll stick with code font :+1: